.github/workflows/tests-pro-integration.yml

```diff
@@ -200,17 +209,16 @@ jobs:
           sudo apt-get update
           # postgresql-14 pin is required to make explicit install of the version from the Ubuntu repos and not PGDG repos
           sudo apt-get install -y --allow-downgrades libsnappy-dev jq postgresql-14=14.11-0ubuntu0* postgresql-client postgresql-plpython3
-
       - name: Cache Ext Dependencies (venv)
         if: inputs.disableCaching != true
         uses: actions/cache@v4
         with:
           path: |
             localstack-ext/.venv
-          # include the matrix group (to re-use the var-libs used in the specific test group)
-          key: community-it-${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-venv-${{ hashFiles('localstack-ext/requirements-test.txt', 'localstack/requirements-test.txt') }}-${{steps.determine-companion-ref.outputs.result}}
+          # include the matrix group (to re-use the venv used in the specific test group)
+          key: community-it-${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-venv-${{ hashFiles('localstack-ext/requirements-test.txt') }}-${{steps.determine-companion-ref.outputs.ref}}-group-${{ matrix.group }}
```

Comment:
Makes sense to remove the community pins as those are not even used in here. Nice catch